.editorconfig

```diff
@@ -10,5 +10,4 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 
 [Dockerfile*]
-indent_style = tab
-indent_size = 8
+indent_size = 4
```

Comment:
reverted this from tab style to space (default)

Reply:
Looks good to me